Crypto/Alchemy/Interpreter/ErrorRateWriter.hs

```diff
@@ -42,7 +42,7 @@ newtype ErrorRateWriter
   a                             -- | represented type
   = ERW { unERW :: k (expr (Monadify w e) (w (Monadify w a))) }
 
-type family Monadify w a = r | r -> a where
```

Comment:
Should the injectivity go back in? (Its removal might have been spurious.) It might be useful later on.

Reply:
Oh yeah, oops. I was testing the effects of removing injectivity and forgot to re-add it.